src/test/java/online/partyrun/partyrunmatchingservice/domain/waiting/root/RunningDistanceTest.java

```diff
@@ -0,0 +1,32 @@
+package online.partyrun.partyrunmatchingservice.domain.waiting.root;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import online.partyrun.partyrunmatchingservice.domain.waiting.exception.NotAllowMeterException;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("RunningDistance")
+class RunningDistanceTest {
+    @Test
+    @DisplayName("거리를 숫자로 입력하면 해당하는 Dinstance 요소를 반환한다")
+    void runGetByMeter() {
+        assertAll(
+                () -> assertThat(RunningDistance.getByMeter(1000)).isEqualTo(RunningDistance.M1000),
+                () -> assertThat(RunningDistance.getByMeter(3000)).isEqualTo(RunningDistance.M3000),
+                () -> assertThat(RunningDistance.getByMeter(5000)).isEqualTo(RunningDistance.M5000),
+                () ->
+                        assertThat(RunningDistance.getByMeter(10000))
+                                .isEqualTo(RunningDistance.M10000));
+    }
```

Comment:
```suggestion
       public static Stream<Arguments> a() {
        return Stream.of(
                Arguments.of(1000, RunningDistance.M1000),
                Arguments.of(3000, RunningDistance.M3000),
                Arguments.of(5000, RunningDistance.M5000),
                Arguments.of(10000, RunningDistance.M10000)
        );
    }

    @ParameterizedTest
    @MethodSource("a")
    @DisplayName("거리를 숫자로 입력하면 해당하는 Dinstance 요소를 반환한다")
    void runGetByMeter(int meter, RunningDistance distance) {
        assertThat(RunningDistance.getByMeter(meter)).isEqualTo(distance);
    }
```

이게 더 뭔가 깔끔해보이는것같은데 어떤가요?
중복되는 로직을 줄일 수 잇네요

Reply:
반영했습니다